RunningOrder/Story/StoryList.swift

```diff
@@ -13,17 +13,19 @@ extension Story: Identifiable {
 }
 
 struct StoryList: View {
-     let stories: [Story]
+    let header: String
+    @Binding var stories: [Story]
+    @EnvironmentObject var toolbarManager: ToolbarManager
 
     var body: some View {
         NavigationView {
             List {
-                Section(header: Text("Stories")) {
+                Section(header: Text(header).font(.headline )) {
```

Comment:
I think the header "Stories" could be above the list (according to the screenshot) instead of just a section header